_clubs/naak/naak-earthcorps-projects-for-2019.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default 
+title: NAAK / EarthCorps Projects for 2019
+date: 2019-07-14 02:25
+author: Klahanie
+comments: true
+categories: [naak]
+---
+
+# NAAK / EarthCorps Projects for 2019
+
+#### Yellow Lake Native Growth Protection Area (NGPA)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Yellow archangel, Purple loosestrife, Tansy ragwort and Garlic mustard.

```

Comment:
"ragwort, and Garlic"

Reply:
Thanks @ALFalls! FYI, a minor tweak to process here, if you can suggest changes using this flow, it'll make it a little faster to incorporate your feedback. Thanks! 😄  https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request